core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java

```diff
@@ -74,8 +74,6 @@
  */
 public abstract class AbstractContextSource implements BaseLdapPathContextSource, InitializingBean {
 
-	private static final String DEFAULT_CONTEXT_FACTORY = "com.sun.jndi.ldap.LdapCtxFactory";
```

Comment:
I think it's preferred to leave this as-is. Several other constants in the class are defined in this way.

Reply:
Done.